app/src/main/java/com/example/myapplication/App.kt

```diff
@@ -1,16 +1,16 @@
 package com.example.myapplication
 
 import android.app.Application
-import android.content.ContentResolver
+import dagger.hilt.android.HiltAndroidApp
 
+@HiltAndroidApp
 class App:Application() {
-    override fun onCreate() {
-        super.onCreate()
-        app = this
-    }
-    companion object {
-        private lateinit var app: App
-
-        val contentResolverInstance: ContentResolver get() = app.contentResolver
-    }
+//    override fun onCreate() {
```

Comment:
Usually not a good practice to push commented blocks. Might wanna look up YAGNI principle. Same goes for data/LocalUsers